src/assessment.js

```diff
@@ -47,7 +47,13 @@ var q2 = [/* your answer here */];
 // context of `alderaan`. In other words, we want to *reference* the `fire`
 // method and temporarily bind it (or, change `this`) to `alderaan` before
 // executing it.
+
+
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  alderaan.fire.bind(alderaan)
```

Comment:
Very close! If you use `bind`, you need to invoke it in order to actually call the function.